_book/03-interactive-notation.md

```diff
@@ -3,16 +3,17 @@ title: "Tutorial 2: Interactive notation"
 
 examples:
     - name: accid-001
-      url: https://raw.githubusercontent.com/rism-digital/verovio.org/gh-pages/_tests/accid/accid-001.mei
+      test-suite: accid/accid-001.mei
       xpath:
         - ".//mei:section/mei:measure[1]"
         - "[...]"
         - ".//mei:section/mei:measure[3]"
     - name: accid-002
-      url: https://raw.githubusercontent.com/rism-digital/verovio.org/gh-pages/_tests/accid/accid-002.mei
+      test-suite: accid/accid-002.mei
       xpath:
         - ".//mei:section/mei:measure[1]"
         - ".//mei:section/mei:measure[2]"
+      options: '{ "transpose": "e" }'
```

Comment:
This could be a dictionary:

```
options:
    transpose: "e"
```

Reply:
Yes, you are right. That would be better. I'll make the change